pom.xml

```diff
@@ -512,6 +509,21 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.owasp</groupId>
+                <artifactId>dependency-check-maven</artifactId>
+                <version>5.0.0-M3</version>
+                <configuration>
+                    <failBuildOnCVSS>4</failBuildOnCVSS>
+                </configuration>
+                <executions>
+                    <execution>
```

Comment:
Can you go ahead and add the suppression file to this PR even if it's blank just so we have something to run with?

Reply:
Looks like @lessarderic added them for the different modules.